### PR TITLE
#927: Selenium ID's

### LINKED ISSUE #927: Create unique ids for certain DOM elements to make automated UI testing more sustainable
I've added TODO comments in my locators.py file, which indicate which DOM elements I would like to have unique ids to make automated testing easier and more sustainable. 

https://github.com/18epedersen/Normandy-e2e-tests/blob/dev/pages/locators.py

### PR DESCRIPTION
**Pull request by andymikulski**
Fixes #927 

- Adds ID's to select elements for use with Selenium tests

---

Regarding ID changes (this is for Emily):

- add a unique id for the recipes card on the home page
  - #gw-recipes-card
- add a unique id for the 'recipes' button on the recipes card
  - #gw-recipes-link
- add a unique id for the new recipe button
  - #lab-recipe-button
  - (also #lab-recipe-link for the a tag)
- add a unique id for the recipe action dropdown
  - Unfortunately, the react component we're using doesnt allow us to add an ID to it.
  - In fact, I don't think it uses a `select` element at all.
  - I've added an ID to a container div, hopefully this suffices: #rf-action-select
- add a unique id for the save button
  - #rf-save-button

- add a unique id or value for the console-log,  show_heartbeat, preference_experiment actions
  - The react component we're using doesn't allow us to add ID to the action options
  - However, you _should_ be able to select them like so: `.ant-select-selection-selected-value[title="console-log"]`

- add a unique id or value for the branch name
  - #pef-branch-name
- add a unique id for the request approval button
  - #dab-request-approval
  - Not to be confused with the `approval request` button (shown when a request is already open)
    - #dab-approval-status
- add a unique id or value for the edit button
  - #dab-edit-button
- add a unique id or value for the clone button
  - #dab-clone-button
- add a unique id the action name on the edit recipe page
  - I think this is a duplicate of the action-related ID's up above

### CODE REVIEW

**Comment by andymikulski** [comment]
@rehandalal @mythmon This is ready for review + merge

File: recipe-server/client/control_new/components/recipes/DetailsActionBar.js [hunk 1]
```diff
@@ -108,28 +108,33 @@ export default class DetailsActionBar extends React.Component {
 
     return (
       <div className="details-action-bar clearfix">
-        <Link href={`${routerPath}/clone`}>
-          <Button icon="swap" type="primary">Clone</Button>
+        <Link href={`${routerPath}/clone`} id="dab-clone-link">
```
Comment (mythmon) [inline recipe-server/client/control_new/components/recipes/DetailsActionBar.js]: dabasaur

**Comment by andymikulski** [comment]
@mythmon @rehandalal Can this PR get a comment when this has been pushed up to dev/staging? @18epedersen is in need of these ID's asap.

**Comment by 18epedersen** [comment]
@andymikulski  Could you guys add a unique ID for the 'add branch' button for a preference-experiment recipe?

Thanks!

**Comment by 18epedersen** [comment]
@andymikulski  Sorry, could you also add unique IDs to the preference values for true and false (this is also after you click 'add branch')?

**Comment by 18epedersen** [comment]
@andymikulski Could you also add unique IDS for the home buttons (on both the breadcrumb and the sidebar?) Thanks!

**Comment by andymikulski** [comment]
@18epedersen You can update #927 with your requests, that's easier to track for us.

**Pull request merged**